src/Telemetry/AzureDevOpsTimeline/AugmentedTimelineIssue.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Internal.AzureDevOps;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline
+{
+    public class AugmentedTimelineIssue
+    {
+        public AugmentedTimelineIssue(int buildId, string timelineId, string recordId, int index, TimelineIssue raw)
+        {
+            BuildId = buildId;
+            TimelineId = timelineId;
+            RecordId = recordId;
+            Index = index;
+            Raw = raw;
+        }
+
+        public int BuildId { get; }
+        public string TimelineId { get; }
+        public string RecordId { get; }
+        public int Index { get; }
+        public TimelineIssue Raw { get; }
```

Comment:
"Raw" is a sort of odd name for this.  I'd probably just call it "Issue" or something.

Reply:
This was existing code. I just refactored it to its own file instead of attached to AzureDevOpsTimeline. I prefer to leave it as is unless you feel strongly about it.

Reply:
Oh sure. Leave it then. :-)